src/index.html

```diff
@@ -5,9 +5,32 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&amp;display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__image" src="images/imac.jpeg"
+      alt="Apple A1419 iMac image">
+      <p class="card__name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__review-val">Reviews: 5</span>
+      </div>
+      <div class="card__price">
+        <span class="card__price_tag">Price:</span>
+        <span class="card__price_val">$2,199</span>
```

Comment:
```suggestion
        <span class="card__price_text">Price:</span>
        <span class="card__price_amount">$2,199</span>
```
I think it is more readable like that